ai_news_generator/main.py

```
@@ -0,0 +1,250 @@
+import os
```

Ruff (0.12.2)

`os` imported but unused

Remove unused import: `os`

(F401)